int16 enc1
int16 enc2